### ISSUE TITLE
BUG - Friends parsing bug when they don't have date of birth

### ISSUE THREAD

**Issue by WestDiscGolf** [aarondcoleman/Fitbit.NET#65]
When a friend hasn't set their date of birth the json response which comes back has the value as `""` which doesn't parse as a valid date time when you request the users friends.

**Comment by aarondcoleman** [comment]
Hmm... should we change it to DateTime? Code ref?

**Comment by WestDiscGolf** [comment]
I've tried setting it to be `DateTime?` however then no values seem to be parsed which is odd.

It's in the inner of ...
```        internal static List<UserProfile> GetFriends(this JsonDotNetSerializer serializer, string friendsJson)
        {
            if (string.IsNullOrWhiteSpace(friendsJson))
            {
                throw new ArgumentNullException("friendsJson", "friendsJson can not be empty, null or whitespace.");
            }

            serializer.RootProperty = "user";
            var friends = JToken.Parse(friendsJson)["friends"];
            return friends.Children().Select(serializer.Deserialize<UserProfile>).ToList();           
        }```
Inside the Deserialize method. I want to say there is a way to configure Json.Net to handle it well but I've not looked into it at the moment.

Also I'm not a fan of changing the model property to `DateTime?` as that will be changing the model from v1 and we are trying to keep it the same to aid with migration to the new version.

**Comment by WestDiscGolf** [comment]
Working on a fix ...

**Comment by WestDiscGolf** [comment]
Fixed in #68

**Comment by WestDiscGolf** [comment]
See PR #70

**Comment by WestDiscGolf** [comment]
Ping :-)

**Comment by mxa0079** [comment]
Fixed in PR #70
